edb/server/http/port.py

```diff
@@ -132,5 +137,25 @@ async def stop(self):
                 for pgcon in self._pgcons_list:
                     pgcon.terminate()
                 self._pgcons_list.clear()
+                if self._http_request_logger is not None:
+                    self._http_request_logger.cancel()
+                    await self._http_request_logger
             finally:
                 await super().stop()
+
+    async def request_stats_logger(self):
+        last_seen = -1
+        while True:
+            current = int(self.last_min_requests)
+            if current != last_seen:
+                telemetry.info(
+                    "HTTP requests for %s-%s in last minute: %d",
+                    self.get_proto_name(),
+                    self._netport,
+                    current,
+                )
+                last_seen = current
+            try:
+                await asyncio.sleep(30)
```

Comment:
30 should be 60? Otherwise the log message is misleading. But if you set it to 60, wouldn't WindowedCounter miss some info?

Reply:
Log message is misleading in any case: there won't be a new log line if the value stays the same from the previous log. So, the minute passed is never from the last log message. It's a minute from the timestamp on the current log message. It still shows a proper 60 second window and as you noticed, will never miss samples.

Reply:
> will never miss samples.

As far as I understand, if we log every 60 seconds we can only potentially miss a second or two of data? In this case sleeping for 60 seconds seems fine.

Reply:
> Log message is misleading in any case: there won't be a new log line if the value stays the same from the previous log

Why is this practical?